src/modules/Blur/Module.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * Blur an Image
+ */
+ module.exports = function Blur(options,UI){
+     options = options || {};
+     options.title = "Blur";
+     options.description = "Blur an Image";
+     options.blur = options.blur || 2
+
+     //Tell the UI that a step has been set up
+     UI.onSetup(options.step);
+     var output;
+
+     function draw(input,callback){
+
+         // Tell the UI that a step is being drawn
+         UI.onDraw(options.step);
+
+         var step = this;
+
+         function extraManipulation(pixels){
+           return pixels = require('ndarray-gaussian-filter')(pixels,options.blur)
+         }
+
+         function changePixel(r, g, b, a){
```

Comment:
You don't have to use this function for this module, this function manipulates the brightness

Reply:
pass a function which returns the current values of rgb

Reply:
so `extraManipulation()` could be renamed `blur()`, right? and then where would that be returned -- on what line? Thanks, @tech4GT !!